packages/@lwc/compiler/src/transformers/__tests__/transform-javascript.spec.ts

```diff
@@ -51,3 +55,84 @@ it('should object spread', async () => {
     expect(code).toContain('b: 1');
     expect(code).not.toContain('...a');
 });
+
+it('should transform unforgeables when Lightning Web Security is on', async () => {
+    const actual = `
+        export const test = window.location.href;
+    `;
+    const { code } = await transform(actual, 'foo.js', {
+        ...TRANSFORMATION_OPTIONS,
+        enableLightningWebSecurityTransforms: true,
+    });
+
+    expect(code).toContain(
+        '(window === globalThis || window === document ? location : window.location).href'
+    );
```

Comment:
☝️ stripWhitespace() calls for these since there are spaces that could become newlines or other space-y things.